include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -308,4 +308,9 @@ static_assert(__cplusplus >= 201703L, "The Range support requires C++17 as minim
 #    define ONEDPL_ALLOW_DEFERRED_WAITING 0
 #endif
 
+//'present' macros
+// shift_left, shift_right; GCC 10; VS 2019 16.1
+#define _ONEDPL_CPP20_SHIFT_LEFT_RIGHT_PRESENT                                                                         \
+    (__cplusplus >= 202002L && ((__GLIBCXX__ && __GLIBCXX__ >= 20200507) || (_MSC_VER && _MSC_VER >= 1921))
```

Comment:
Still the check for libstdc++ is not correct

Reply:
Why? I've checked it with GodBolt...

Reply:
Because it just a date. Date is not robust enough. User may have its own Gnu toolchain with any dates. (for example built from source). Some branches may go forward even after the official release. (we basically may have the same approach for oneDPL).

There is another macro that checks the version of libstdc++ as I wrote you. GLIBCXX_RELEASE or something like that. You may see how it's already used in onedpl_config.h